libs/core/src/lib/formatted-text/utils/html-sanitizer.ts

```diff
@@ -0,0 +1,200 @@
+interface StringKey<T> {
+    [key: string]: T;
+}
+
+interface SanitizeWrapper {
+    iframe: HTMLIFrameElement | null;
+    iframeDoc: Document | null;
+}
+
+export class HtmlSanitizer {
+    tagWhitelist: StringKey<boolean> = {};
+    attributeWhitelist: StringKey<string | boolean | Function> = {};
+
+    private schemaWhiteList: string[] = ['http', 'https', 'ftp', 'mailto'];
+
+    private uriAttributes: StringKey<boolean> = {
+        href: true
+    };
+
+    private safeWrapper?: SanitizeWrapper = {
+        iframe: null,
+        iframeDoc: null
+    };
+
+    constructor() {
+        this.extendTags();
+    }
+
+    get defTagWhitelist(): StringKey<boolean> {
+        return {
+            A: true,
+            ABBR: true,
+            BLOCKQUOTE: true,
+            BR: true,
+            CITE: true,
+            CODE: true,
+            DL: true,
+            DT: true,
+            DD: true,
+            EM: true,
+            H1: true,
+            H2: true,
+            H3: true,
+            H4: true,
+            H5: true,
+            H6: true,
+            P: true,
+            PRE: true,
+            STRONG: true,
+            SPAN: true,
+            U: true,
+            UL: true,
+            OL: true,
+            LI: true
+        };
+    }
+
+    get defAttributeWhitelist(): StringKey<string | boolean | Function> {
+        return {
+            class: true,
+            style: true,
+            href: true,
+            target: '_blank',
+            download: true,
+            hreflang: true,
+            rel: true,
+            type: true,
+            title: true
+        };
+    }
+
+    extendTags(customTags?: StringKey<boolean | string>): void {
+        this.tagWhitelist = { ...this.defTagWhitelist, ...customTags, BODY: true };
+    }
+
+    extendAttrs(customAttrs?: StringKey<boolean | string>): void {
+        this.attributeWhitelist = { ...this.defAttributeWhitelist, ...customAttrs };
+    }
+
+    sanitizeHtml(input: string): string {
+        input = input.trim();
+        if (input.length === 0) {
+            return '';
+        }
+
+        if (input === '<br>') {
+            return '';
+        }
```

Comment:
It's the same as an empty line. I think that it's not a goal for this component to render an empty line. But you can use `br` tag in your embed code as a separator.

Reply:
Yeah, I see that there is no much sense in rendering one `br`.
The documentation says that `br` is allowed so I would consider it as unexpected behaviour in case when it's not rendered. App developer should decide if he/she wants to render only one br or not.
SO I believe we should rendered allowed tags.

Reply:
Agree, I'll remove this condition.

p.s.
@InnaAtanasova 
@dimamarksman 
Sorry, I marked conversations with "Fixed" comment before pushed all changes :) It's only for my side, and I will resend for review after pushed my changes. And reviewers can resolve conversations if it's fixed.
Is it a normal flow?